docs/ybm_cluster_encryption_update-state.md

```diff
@@ -0,0 +1,37 @@
+## ybm cluster encryption update-state
+
+Update Encryption at Rest (EaR) state for a cluster
+
+### Synopsis
+
+Update Encryption at Rest (EaR) state for a cluster
+
+```
+ybm cluster encryption update-state [flags]
+```
+
+### Options
+
+```
+      --enable   [REQUIRED] Whether to enable or disable EAR (default true)
+  -h, --help     help for update-state
+```
+
+### Options inherited from parent commands
+
+```
+  -a, --apiKey string         YBM Api Key
```

Comment:
@cdavid As discussed offline, it would be good to have a ybm configure command (similar to aws configure) where we can get the api key and store the account and project details from the API key information locally (~/.ybm-cli/xx)

Reply:
This already exist with ybm auth no ? this options is just if you want to override ..

Reply:
Thanks for the update. 
Do we need this to be overridden though?

https://docs.aws.amazon.com/cli/latest/reference/ec2/describe-instances.html

Does not allow the auth to be overridden for every command

Reply:
Actually this is not true, you can specify AWS KEY, using environment variable, so you can override for every command if you want too.  Happy to remove that if @shreyas-agnihotri agree.

Reply:
Agreed, but there is no command line option to override. Perhaps we can think along the same lines since using ENV variables seems common practice

Reply:
Please file a JIRA for this

Reply:
We already support env variable for this as well,  env variable, command line, and file. 

You can create a JIRA ticket if you want, I am not a SE so I can't really create them

Reply:
@kc-yb - not sure what you are asking for here.

We support the following:
`ybm auth` - stores the api key and URL and basic configs.
`ybm <command>` - read from config file
`ybm <command> --config <path to config file>`
`ybm <command> --apiKey <ybm key>`
`YBM_CONFIG=<config file path> ybm <command>`
`YBM_APIKEY=<ybm key> ybm <command>`

The priority is `default config file < environment variables < command flags`.

Did you have something else in mind?

Reply:
I did not know we had added this feature.

Also do we need an override for api key in every command - env variable and ybm auth should suffice I think